src/main/java/longtimenosee/logic/parser/PolicyAssignedDeleteCommandParser.java

```diff
@@ -21,9 +21,15 @@ public PolicyDeleteAssignedCommand parse(String args) throws ParseException {
         Index personIndex;
         Index policyIndex;
 
+        String trimmedInput = args.trim();
+        String[] indexes = (trimmedInput.split(" "));
+
+        if (indexes.length != 2) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    PolicyDeleteAssignedCommand.MESSAGE_USAGE));
+        }
+
         try {
-            String trimmedInput = args.trim();
```

Comment:
![image](https://user-images.githubusercontent.com/97422591/199454862-1620e8f4-4b1a-495e-871c-a2b5561a8b07.png)

I like how it states both client and policy, good job. As for the "from Client: ", not sure if it's intentional but there's a space. I think it looks better with the space though haha. I'll leave it up to you to decide.